extensions/todoist/src/helpers/dates.ts

```diff
@@ -63,9 +64,20 @@ export function displayDate(dateString: string) {
   return format(date, "dd MMMM yyy");
 }
 
-export function displayDateTime(dateString: string) {
-  const date = displayDate(dateString);
-  return `${date} ${format(new Date(dateString), "HH:mm")}`;
+export function DisplayDateTime(dateString: string) {
+  const date = parseISO(dateString);
+  const [data] = useCachedData();
+  const use12HourFormat = data?.user?.time_format === 1;
+
+  return `${displayDate(dateString)} ${format(date, use12HourFormat ? "h:mm a" : "HH:mm")}`;
+}
+
+export function DisplayTime(dateString: string) {
+  const date = parseISO(dateString);
+  const [data] = useCachedData();
+  const use12HourFormat = data?.user?.time_format === 1;
+
+  return format(date, use12HourFormat ? "h:mm a" : "HH:mm");
 }
```

Comment:
Could you pass the setting as a parameter instead? Calling a hook inside a function that’s not a hook or a React component breaks the rules of hooks.

Reply:
Also, a small nit: is it possible to make the function names lowercase instead to avoid any confusion?